Vmc 358 modified

prabeam opened the pull request.

prabeam committed:
VMC-358 Added new analysis tab and added code to retain analysis tab on
page refresh.

sunilmanandhar committed:
VMC-358 Added HTML Design on the newly added "Analysis tab" according to the Bmockup design format similar to schema tab design.

sunilmanandhar committed:
VMC-358 Added HTML Design on the newly added "Analysis tab" according to the Mockup-A design format.

prabeam committed:
VMC-358 Added tabs and sub tabs related to Analysis Page.

sujmanshak committed:
VMC-358: fetching data from @ProcedureDetail API for Analysis Tab

sujmanshak committed:
Merge branch 'VMC-358' of http://github.com/deerwalk/voltdb into VMC-358

prabeam committed:
VMC-358 Modified code to add sample latency graph to latency tab in Analysis
page.

prabeam committed:
Merge branch 'VMC-358' of https://github.com/deerwalk/voltdb into VMC-358

sujmanshak committed:
VMC-358: Fetching data from API to plot in analysis procedures tab

sujmanshak committed:
Merge branch 'VMC-358' of http://github.com/deerwalk/voltdb into VMC-358

Conflicts:
	src/frontend/org/voltdb/dbmonitor/index.htm

prabeam committed:
VMC-358 Modified code to add click event to each bars in latency chart
of Analysis page.

prabeam committed:
VMC-358 Modified code to show actual data in latency chart of analysis page
and added frequency and combined value in tool-tip.

sujmanshak committed:
VMC-358: Working on Latency Tables popup

sujmanshak committed:
Merge branch 'VMC-358' of http://github.com/deerwalk/voltdb into VMC-358

prabeam committed:
VMC-358 Modified code to add frequency and combined charts, change layout
tool tips and make Procedure tab as default tab in Analysis page.

sujmanshak committed:
VMC-358: Design completed for detailed view of latency for particular procedures

sujmanshak committed:
Merge branch 'VMC-358' of http://github.com/deerwalk/voltdb into VMC-358

prabeam committed:
VMC-358 Renamed Latency to Execution time and modified code changed the
legend title while changing procedure sub tabs.

prabeam committed:
VMC-358 Modified code to fix issue with graph's legend and extra controls
in Analysis page.

prabeam committed:
VMC-358 Fixed the issue with "no data available" message and graph being
shown at same time.

sujmanshak committed:
VMC-358: displaying @ProcedureDetail API data in latency details popup

sujmanshak committed:
Merge branch 'VMC-358' of http://github.com/deerwalk/voltdb into VMC-358

Conflicts:
	src/frontend/org/voltdb/dbmonitor/js/voltdb.graph.js

prabeam committed:
VMC-358 Modified code to fix issue with fractional digits not being shown
properly in Analysis charts.

prabeam committed:
VMC-358 Modified code to add procedure chart generated date.

sujmanshak committed:
VMC-358: Displaying dynamic popups for latency details

sujmanshak committed:
Merge branch 'VMC-358' of http://github.com/deerwalk/voltdb into VMC-358

Conflicts:
	src/frontend/org/voltdb/dbmonitor/css/layout.css
	src/frontend/org/voltdb/dbmonitor/js/voltdb.analysis.js

prabeam committed:
VMC-358 Modified code to fix issue with latency chart not being re-sized
on page re-sized.

prabeam committed:
Merge branch 'VMC-358' of https://github.com/deerwalk/voltdb into VMC-358

Conflicts:
	src/frontend/org/voltdb/dbmonitor/js/voltdb.graph.js

sujmanshak committed:
VMC-358: Added realtime timestamp value from procedure detail API

sujmanshak committed:
Merge branch 'VMC-358' of http://github.com/deerwalk/voltdb into VMC-358

prabeam committed:
VMC-358 Modified code to fix issue with chart's click event on page
refresh.

prabeam committed:
Merge branch 'VMC-358' of https://github.com/deerwalk/voltdb into VMC-358

sujmanshak committed:
VMC-358: tooltip added for Latency Details graph

sujmanshak committed:
Merge branch 'VMC-358' of http://github.com/deerwalk/voltdb into VMC-358

Conflicts:
	src/frontend/org/voltdb/dbmonitor/js/voltdb.graph.js

sujmanshak committed:
VMC-358: Fixed issue with latency popup showing multiple times

prabeam committed:
VMC-358 Fixed issue with long procedure name not being shown properly in
charts. Also added units in tool-tips.

prabeam committed:
Merge branch 'VMC-358' of https://github.com/deerwalk/voltdb into VMC-358

Conflicts:
	src/frontend/org/voltdb/dbmonitor/js/nv.d3.min(modified).js
	src/frontend/org/voltdb/dbmonitor/js/voltdb.graph.js

sujmanshak committed:
VMC-358: Fixed issue with procedure Details graph not plotting properly

sujmanshak committed:
VMC-358: Fixed issue with Procedure Name not showing properly in latency detail pop up

sujmanshak committed:
VMC-358: Fixed issue with SqlQuery tab flickering when navigated from Analysis tab

prabeam committed:
VMC-358 Modified code to show loading image while analysis data is being
loaded.

prabeam committed:
Merge branch 'VMC-358' of https://github.com/deerwalk/voltdb into VMC-358

Conflicts:
	src/frontend/org/voltdb/dbmonitor/js/voltdb.analysis.js

prabeam committed:
VMC-358 Modified code to rename the procedure name label.

sujmanshak committed:
VMC-358: added designs for setting Thesholds and Analysis Remarks

sujmanshak committed:
Merge branch 'VMC-358' of http://github.com/deerwalk/voltdb into VMC-358

prabeam committed:
VMC-358 Modified code to change color of bar chart for multi-partitioned
procedures and display warning sign for procedures with combined usage value
greater than 20%. Also added procedure type field in tool-tips.

prabeam committed:
Merge branch 'VMC-358' of https://github.com/deerwalk/voltdb into VMC-358

prabeam committed:
VMC-358 Modified code to fix issue with warning sign not being displayed
in Firefox browser. Also removed debuggers.

prabeam committed:
VMC-358 Modified code to change legend for multi partitioned procedure.

sujmanshak committed:
VMC-358: Fixed various design and spelling mistakes in Latency Details popup

sujmanshak committed:
Merge branch 'VMC-358' of http://github.com/deerwalk/voltdb into VMC-358

sujmanshak committed:
VMC-358: Fixed design issue in Latency Details graph in firefox browser

prabeam committed:
VMC-358 Modified code to fix the position of warning sign in charts.
Modified code to get partition type of procedure.

prabeam committed:
Merge branch 'VMC-358' of https://github.com/deerwalk/voltdb into VMC-358

prabeam committed:
VMC-358 Modified code to change pointer type while hovering over bar chart.

prabeam committed:
VMC-358 Modified code to show procedure type properly and show warning
messages in tool-tips and "Analysis Remark" section of page.

prabeam committed:
VMC-358 Modified code to change legend title and color of warning text.

prabeam committed:
VMC-358 Fixed design issues related to analysis tab on different
resolution for mobile and iPad.

prabeam committed:
VMC-358 Modified code to fix the issue with height of Analysis page in mobile
devices.

prabeam committed:
VMC-358 Modified code to fix chart size for different resolutions.

prabeam committed:
VMC-358 Modified code to fix design issues with procedure pop-up.
Also modified code to fix alignment issue on analysis chart.

prabeam committed:
VMC-358 Modified code to fix issue with bar chart height being randomly
increased on tab change.

prabeam committed:
VMC-358 Modified code to fix chart labels in mobile.

prabeam committed:
VMC-358 Modified code to change decimal format of combined usage value in
analysis chart.

sujmanshak committed:
VMC-358: Added feature to set Threshold for usage percentage manually

sujmanshak committed:
VMC-358: Renamed Threshold terminologies and fixed design issues

prabeam committed:
VMC-358 Modified code to fix position of warning sign.

sujmanshak committed:
VMC-358: Added analysis message for frequency threshold

sujmanshak committed:
Merge branch 'VMC-358' of http://github.com/deerwalk/voltdb into VMC-358

sujmanshak committed:
VMC-358: Fixed issue with latency Detail popup width issue

sujmanshak committed:
VMC-358: added Frequency Details pop up in Frequency Procedure graph

sujmanshak committed:
VMC-358: Added Combined Details popup

akhanzode committed:
Merge remote-tracking branch 'origin/master' into VMC-358

akhanzode committed:
Merge remote-tracking branch 'origin/master' into VMC-358

prabeam committed:
VMC-358 Modified code to hide Data tab from Analysis tab.

sujmanshak committed:
VMC-358-modified: changed logic to display Average Execution Time in latency Details popup

prabeam committed:
VMC-358-modified: Removed partition id from tool-tip in Execution Time
drill down pop-up.

sujmanshak committed:
VMC-358-modified: Fixed issue with Frequency drill down chart not plotting properly

sujmanshak committed:
Merge branch 'VMC-358-modified' of http://github.com/deerwalk/voltdb into VMC-358-modified

prabeam committed:
VMC-358-modified: Modified code to hide scroll bar from charts in
Analysis tab when not needed.

prabeam committed:
Merge branch 'VMC-358-modified' of https://github.com/deerwalk/voltdb into VMC-358-modified

prabeam committed:
VMC-358-modified: Modified code to fix issue with bar size in drill
down chart.

sujmanshak committed:
VMC-358-modified: Changed logic for displaying drill down chart for combined graph

sujmanshak committed:
Merge branch 'VMC-358-modified' of http://github.com/deerwalk/voltdb into VMC-358-modified

prabeam committed:
VMC-358-modified: Modified code to re-size drill down charts when pop-up is
re-sized.

prabeam committed:
VMC-358-modified: Modified code to show detail values in tool-tip for
drill-down frequency and combined chart.

prabeam committed:
VMC-358-modified: Modified code to fix issue of warning message being
displayed for single partitioned procedure also.

prabeam committed:
VMC-358-modified: Modified code to change labels, value formats and units
in tool-tips of drill-down charts and also hide frequency warning message
in analysis page when frequency threshold is not set.

sujmanshak committed:
VMC-358-modified: Fixed issue with X-axis overlapping with the popup in drilldown charts

sujmanshak committed:
Merge branch 'VMC-358-modified' of http://github.com/deerwalk/voltdb into VMC-358-modified

sujmanshak committed:
VMC-358-modified: Fixed issue with Tooltip not showing in Execution Time drill down chart

sujmanshak committed:
VMC-358-modified: Fixed issue with wrong count showing in frequency drill down chart

prabeam committed:
VMC-358-modified: Modified code to remove index no. from tool-tip of drill-down chart.

prabeam committed:
Merge branch 'VMC-358-modified' of https://github.com/deerwalk/voltdb into VMC-358-modified

sujmanshak committed:
VMC-358-modified: Removed <All> Statement from drill down charts

prabeam committed:
VMC-358-modified: Modified code to change legend for execution time and fix
the issue of modified date being hidden.

prabeam committed:
Merge branch 'VMC-358-modified' of https://github.com/deerwalk/voltdb into VMC-358-modified

prabeam committed:
VMC-358-modified: Modified code to rename "Combined" to "Total Processing
Time" in Analysis tab.

sujmanshak committed:
VMC-358-modified: Changed logic to get Min, Max and invocations for drill down charts

sujmanshak committed:
Merge branch 'VMC-358-modified' of http://github.com/deerwalk/voltdb into VMC-358-modified

Conflicts:
	src/frontend/org/voltdb/dbmonitor/js/nv.d3.min(modified).js

prabeam committed:
VMC-358-modified: Modified code to rename pop-up tile to "Settings" and add
message bar in Setting pop-up. Also modified code to remove frequency and
add average execution time with default value of 500ms in setting pop-up.

prabeam committed:
Merge branch 'VMC-358-modified' of https://github.com/deerwalk/voltdb into VMC-358-modified

prabeam committed:
VMC-358-modified: Modified code to display warning messages for both single
and multi partitioned procedures. Also modified code to change warning
messages.

prabeam committed:
VMC-358-modified: Modified methods to calculate and display total processing
time in chart.

sujmanshak committed:
VMC-358-modified: Changed formula to calculate Total Processing Time

sujmanshak committed:
Merge branch 'VMC-358-modified' of http://github.com/deerwalk/voltdb into VMC-358-modified

Conflicts:
	src/frontend/org/voltdb/dbmonitor/js/nv.d3.min(modified).js

sujmanshak committed:
VMC-358-modified: added calculation for MP procedure in Total Processing time details chart

prabeam committed:
VMC-358-modified: Modified code to add setting for "Show system procedure".

prabeam committed:
Merge branch 'VMC-358-modified' of https://github.com/deerwalk/voltdb into VMC-358-modified

sujmanshak committed:
VMC-358-modified: Fixed issue with invocations not displaying properly in tooltip

sujmanshak committed:
VMC-358-modified: fixed issue with legend showing when there is no data in drill down chart

sujmanshak committed:
Merge branch 'VMC-358-modified' of http://github.com/deerwalk/voltdb into VMC-358-modified

prabeam committed:
VMC-358-modified: Modified code to change decimal format of data in
frequency and combined charts.

prabeam committed:
Merge branch 'VMC-358-modified' of https://github.com/deerwalk/voltdb into VMC-358-modified

sujmanshak committed:
VMC-358-modified: Fixed issue with total invocations showing wrong count in drill down charts

sujmanshak committed:
Merge branch 'VMC-358-modified' of http://github.com/deerwalk/voltdb into VMC-358-modified

sujmanshak committed:
VMC-358-modified: Fixed design issue with drill down chart when no data is available

akhanzode committed:
Merge remote-tracking branch 'upstream/master' into VMC-358-modified

prabeam committed:
VMC-358-modified: Modified code to make bar widths fixed in Analysis charts.

prabeam committed:
Merge branch 'VMC-358-modified' of https://github.com/deerwalk/voltdb into VMC-358-modified

prabeam committed:
VMC-358-modified: Modified code to remove 1,2, 3, 4 from the main chart
labels.

prabeam committed:
VMC-358-modified: Modified code to remove 1,2, 3, 4 from the drill down
chart labels of total processing time and frequency.

prabeam committed:
VMC-358-modified: Modified code to fix issue with drill down pop-up not being
shown on page refresh.

akhanzode committed:
Merge remote-tracking branch 'upstream/master' into VMC-358-modified

sujmanshak committed:
VMC-358-modified: Fixed issue with Total Processing Time drill down not showing

sujmanshak committed:
VMC-358-modified: Modified code to remove 1,2, 3, 4 from the Total Processing Time drill down chart

sujmanshak committed:
VMC-358-modified: Fixed margin issue with Total Processing Time drill down chart

sujmanshak committed:
VMC-358-Modified: Fixed issue with invocations calculations in Multi Partition

sujmanshak committed:
VMC-358-modified: Fixed issue with Total Processing time drill down chart not drawing properly

prabeam committed:
VMC-358-modified-all: Modified code to replace data from procedure profile
API with that from procedure detail API

prabeam committed:
Merge branch 'VMC-358-modified-all' into VMC-358-modified

prabeam committed:
VMC-358-modified: Modified code to make "Show system procedures" unchecked
by default and change legend of frequency chart to "Number Of Invocations".

prabeam committed:
VMC-358-modified: Modified code to change the scale of the Total Execution
Time graph to seconds and remove unwanted functions.

prabeam committed:
VMC-358-modified:Modified code to remove threshold for total processing
time and change the explanatory text in Settings pop-up.

prabeam committed:
VMC-358-modified: Modified code to add unit markers to the labels on the
bars.

prabeam committed:
VMC-358-modified: Modified code to sort analysis charts properly.

prabeam committed:
VMC-358-modified: Modified code to fix issue with system procedure warnings
being displayed when system procedure is unchecked.

prabeam committed:
VMC-358-modified: Modified code to order drill down chart properly and change
units in it.

prabeam committed:
VMC-358-modified: Modified legend for drill down charts.

iarlina committed:
Changed Number of Invocation to Number of Invocations

akhanzode committed:
Merge remote-tracking branch 'upstream/master' into VMC-358-modified

akhanzode committed:
Merge branch 'VMC-358-modified' of github.com:deerwalk/voltdb into VMC-358-modified

akhanzode committed:
Merge remote-tracking branch 'origin/master' into VMC-358-modified

akhanzode committed:
Merge remote-tracking branch 'upstream/master' into VMC-358-modified

akhanzode committed:
Merge remote-tracking branch 'origin/master' into VMC-358-modified

akhanzode committed:
Merge remote-tracking branch 'origin/master' into VMC-358-modified

akhanzode commented on src/frontend/org/voltdb/dbmonitor/js/voltdb.ui.js:
```diff
@@ -2800,6 +3107,14 @@ var adjustImporterGraphSpacing = function() {
             }
         };
 
+//        this.calculateCombinedDetailValue=function(profileData){
```
Can you remove commented code?

akhanzode reviewed (changes requested):
Do you have any GEB tests for this page?

prabeam committed:
VMC-358-modified: Added Geb test files related to analysis page.
Also added a basic test to check if Analysis page is opening properly.

prabeam committed:
VMC-358-modified: Added geb test to check if Analyze button is working
properly. Also added tests to check if procedure sub tabs are displayed
properly when tables and procedures are added in SQL page.

prabeam committed:
VMC-358-modified: Added geb tests to check if graphs are displayed in
procedure sub tabs. Also added tests to check if setting pop-up is working
properly.

akhanzode committed:
Merge remote-tracking branch 'origin/master' into VMC-358-modified

iarlina committed:
VMC-358-modified: Added geb test for analysis tab

prabeam committed:
VMC-358-modified: Removed unwanted commented codes.

akhanzode committed:
Merge remote-tracking branch 'origin/master' into VMC-358-modified
